python/ray/serve/router.py

```diff
@@ -76,17 +86,14 @@ def __init__(
         # the same node.
         self.replica_iterator = itertools.cycle(self.in_flight_queries.keys())
 
-        # Used to unblock this replica set waiting for free replicas. A newly
-        # added replica or updated max_concurrent_queries value means the
-        # query that waits on a free replica might be unblocked on.
-        self.config_updated_event = asyncio.Event(loop=event_loop)
+        self.pending_requests: Deque[PendingQuery] = deque()
```

Comment:
inconsistent naming here. let's pick either "query" or "request" and stick with it